docs/prepare_prior_versions.py

```diff
@@ -55,7 +55,22 @@ def _paths_to_versioned_docs_after_v0_15_50() -> list[pathlib.Path]:
     paths = [
         f
         for f in data_path.iterdir()
-        if f.is_dir() and ("0.14.13" not in str(f) or "0.15.50" not in str(f))
```

Comment:
the or here looks like it was just making this return True for every version AFAICT

Reply:
What is the significance of these strings? Could we make them constants to give them some more context?

Reply:
They're past versions. We could make them constants, but I think it'd still be fragile. What if we just plan to fix this in https://greatexpectations.atlassian.net/browse/V1-131?